pkg/registry/core/rest/storage_core.go

```diff
@@ -213,6 +213,7 @@ func (c LegacyRESTStorageProvider) NewLegacyRESTStorage(apiResourceConfigSource
 	if err != nil {
 		return LegacyRESTStorage{}, genericapiserver.APIGroupInfo{}, fmt.Errorf("cannot create cluster IP allocator: %v", err)
 	}
+	serviceClusterIPAllocator.EnableMetrics()
 	restStorage.ServiceClusterIPAllocator = serviceClusterIPRegistry
```

Comment:
I have to remember how this ends in the repair loops

https://github.com/kubernetes/kubernetes/blob/9d85e18ec0dc09f681650d04d65e3db5237bef96/pkg/controlplane/controller.go#L174

Reply:
tracing

https://github.com/kubernetes/kubernetes/blob/9d85e18ec0dc09f681650d04d65e3db5237bef96/pkg/controlplane/instance.go#L522-L548